prisma/schema.prisma

```diff
@@ -8,6 +8,18 @@ generator client {
 }
 
 model User {
-  id    Int    @id @default(autoincrement())
-  email String
+  id     Int      @id @default(autoincrement())
+  email  String
+  Review Review[]
+}
+
+model Review {
+  id         Int      @id @default(autoincrement())
+  movie_id   String
+  author_id  Int
+  content    String?
```

Comment:
Cool, so a Review can be just rating only right?

Reply:
Correct...no requirement to actually add text